README.md

```diff
@@ -35,21 +35,29 @@ For more information please review the [guidelines for contributing](https://git
 So you would like to contribute by testing? Awesome, we appreciate that very much. 
 
 To report a bug for the alpha or beta version, just create an issue on github like you would for the stable version and
- provide the version number. 
-In addition to google play, the alpha and beta apps can also be obtained from the Nextcloud [Download page](https://download.nextcloud.com/android/talk-alpha/). Please remember that Google Services are necessary to receive push notifications. 
+ provide the version number. Please remember that Google Services are necessary to receive push notifications. 
  
-#### Beta Channel
+#### Beta versions (Release Candidates)
 
+##### via Google Play
 Sign up at [Google Play Beta channel](https://play.google.com/apps/testing/com.nextcloud.talk2) to get Release
  Candidates via Google Play.
 
-#### Alpha Channel
+##### via github
+You can also get the Release Candidates at [github releases](https://github.com/nextcloud/talk-android/releases).
 
+#### Alpha versions
+
+##### via Google Play
 To become an alpha tester you have to be signed up for the [Google Play Beta channel](https://play.google.com/apps/testing/com.nextcloud.talk2) 
 and additionally you have to join the [Alpha testing Google Group](https://groups.google.com/g/nextcloud-android-talk-alpha-testing). 
 After that you will receive the alpha versions via the Play Store (initially, this might take some minutes after
- signing up).
-Alpha versions are delivered weekly. Please make sure to remember that these versions might contain bugs and you don't use them in production.
+ signing up). However, in in the Play Store the app will still be named "Nextcloud Talk (Beta)" even if you are an alpha tester, but you will receive the alpha versions.
```

Comment:
"in in"

Reply:
@tobiasKaminsky You caught it nicely 😉.

Reply:
@mahibi It remained to fix and Merge 😉.

> signing up). However, in **in** the Play Store the app will still be named "Nextcloud Talk (Beta)" even if you are an alpha tester, but you will receive the alpha versions.

Reply:
@Valdnet "in in" is fixed. Here in the conversations view you still see the old changes (so the "Outdated" label is shown). In the "Files changed" view it is fixed